File: problems/gcd.zig
const std = @import("std");

fn GCD(a: u32, b: u32) u32{
    if(b == 0)
        return a;
    return GCD(b, a%b);
}

pub fn main() !void{
    const a: u32 = 48;
    const b: u32 = 18;
    std.debug.print("GCD({}, {}) = {}\n", .{a,b,GCD(a,b)});
}

File: problems/gcd.c
#include <stdio.h>

int GCD(int a, int b)
{
  if(b==0)
    return a;
  return GCD(b, a%b);
}

int main(){
  int a = 18, b = 48;
  printf("GCD(%d, %d) = %d\n", a, b, GCD(a,b));
  return 0;
}
